extensions/qute-resteasy/deployment/src/test/java/io/quarkus/qute/resteasy/deployment/ItemResource.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.qute.resteasy.deployment;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import io.quarkus.qute.Rendering;
+import io.quarkus.qute.api.VariantTemplate;
+
+@Path("item")
+public class ItemResource {
+
+    @Inject
+    VariantTemplate item;
```

Comment:
I'm not sure why we need to make variants explicit here. It requires to modify the controller when we could just add a new template file and it could be picked up as a variant. The fact that more than one version exists should be enough to guide the implementation to know there are variants.